Resources/doc/sync/descriptors/descriptors.rst

```diff
@@ -0,0 +1,122 @@
+Descriptors
+===========
+
+Extraction descriptors are special classes that tell extractor which fields/tables to watch for changes.
+
+By default, all services tagged with a ``{ name: ongr_connections.extraction_descriptor }`` are added to a single service
+called ``ongr_connections.sync.extraction_collection``, which later on can be passed as a parameter for
+an `extractor <../extractor/extractor.rst>`_ via ``setExtractionCollection`` method.
+
+When `extractor <../extractor/extractor.rst>`_ looks for changes, it iterates through the descriptors -
+executing each descriptor as to mark down which changes in the database are relevant.
+
+    NOTE: Only ``DoctrineExtractor`` supports descriptors so far.
+
+Example extractor configuration:
+
+.. code-block:: yaml
+
+    # Extractor service.
```

Comment:
but this service, is default, it wont be overriden by users, right? if so, we don't need any examples